cfme/containers/provider/openshift.py

```diff
@@ -113,6 +113,12 @@ def __init__(
 
     def create(self, **kwargs):
 
+        # Enable C&U for any provider with metrics provider
+        if getattr(self, "metrics_type") != "Disabled":
+            server = self.appliance.server.settings
```

Comment:
`server` is a bit of a mis-name here, I'd rather just see a direct call to `self.appliance.server.settings.enable_server_roles()` since its not used for any further calls.

Reply:
fixed....